pull-requests/README.md

```diff
@@ -19,10 +20,12 @@
 
 |               Route                |           Description           |
 | :--------------------------------: | :-----------------------------: |
-|      [GET /pullrequests/:id](#get-pullrequestsid)      | Returns latest PRs by the user in RDS |
+|      [GET /pullrequests/user/:id](#get-pullrequestsusserid)      | Returns latest PRs by the user in RDS |
+|      [GET /pullrequests/open](#get-pullrequestsopen)      | Returns 10 latest open PRs in RDS |
+|      [GET /pullrequests/stale](#get-pullrequestsstale)       | Returns stale PRs in RDS |
 
 
-## **GET /pullrequests/:id**
+## **GET /pullrequests/user/:id**
```

Comment:
One small change please: we are not passing `id` anymore. We are passing `username` now.
Can you please change it at the places it is needed? 

Also on line 23, when I click on the title, it's not taking me to the required id. You can verify that using either online Markdown editors or extension for VSCode or in Github preview as well. 

I know I was supposed to do it 😅
Let me know if that's feasible or I'll do it.

Reply:
No problem @swarajpure . Its done